[2.0] NPE on ReflectionUtils

Having an endpoint:
```yaml
  /fake/inline-additionalProperties:
    post:
      tags:
      - fake
      summary: test inline additionalProperties
      operationId: testInlineAdditionalProperties
      requestBody:
        description: request body
        content:
          application/json:
            schema:
              type: object
              additionalProperties:
                type: string
        required: true
      responses:
        200:
          description: successful operation
          content: {}
```
will throw a NPE once the inflector project is running with

```
mvn jetty:run
```

```console
Caused by: java.lang.NullPointerException
    at io.swagger.oas.inflector.utils.ReflectionUtils.getTypeFromProperty (ReflectionUtils.java:192)
    at io.swagger.oas.inflector.utils.ReflectionUtils.getTypeFromProperty (ReflectionUtils.java:195)
    at io.swagger.oas.inflector.utils.ReflectionUtils.getTypeFromModel (ReflectionUtils.java:335)
    at io.swagger.oas.inflector.utils.ReflectionUtils.getTypeFromRequestBody (ReflectionUtils.java:145)
    at io.swagger.oas.inflector.utils.ReflectionUtils.getOperationParameterClasses (ReflectionUtils.java:94)
    at io.swagger.oas.inflector.controllers.OpenAPIOperationController.detectMethod (OpenAPIOperationController.java:180)
    at io.swagger.oas.inflector.controllers.OpenAPIOperationController.<init> (OpenAPIOperationController.java:141)
    at io.swagger.oas.inflector.OpenAPIInflector.createController (OpenAPIInflector.java:457)
    at io.swagger.oas.inflector.OpenAPIInflector.addOperation (OpenAPIInflector.java:444)
    at io.swagger.oas.inflector.OpenAPIInflector.init (OpenAPIInflector.java:167)
    at io.swagger.oas.inflector.OpenAPIInflector.<init> (OpenAPIInflector.java:108)
```

--- a/src/main/java/io/swagger/oas/inflector/utils/ReflectionUtils.java
+++ b/src/main/java/io/swagger/oas/inflector/utils/ReflectionUtils.java
@@ -18,6 +18,7 @@ package io.swagger.oas.inflector.utils;
 
 import com.fasterxml.jackson.databind.JavaType;
 import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.type.SimpleType;
 import com.fasterxml.jackson.databind.type.TypeFactory;
 import io.swagger.oas.inflector.Constants;
 import io.swagger.oas.inflector.config.Configuration;
@@ -191,7 +192,7 @@ public class ReflectionUtils {
         if(property.getAdditionalProperties() != null && property.getAdditionalProperties() instanceof Schema) {
             Schema inner = (Schema)property.getAdditionalProperties();
             JavaType innerType = getTypeFromProperty(null, null, inner, definitions);
-            return tf.constructMapLikeType(Map.class, getTypeFromProperty("string", null, null, definitions), innerType);
+            return tf.constructMapLikeType(Map.class, SimpleType.constructUnsafe(String.class), innerType);
         }
         if(property.get$ref() != null) {
             if(definitions != null) {

